recipes-debian/sysvinit/sysvinit_debian.bb

```diff
@@ -13,7 +13,8 @@ inherit update-alternatives distro_features_check debian-package
 require recipes-debian/sources/sysvinit.inc
 
 FILESPATH_append = ":${COREBASE}/meta/recipes-core/sysvinit/sysvinit:"
-SRC_URI += "file://install.patch \
+SRC_URI += "file://build-mountpoint.patch \
```

Comment:
We should use `EXTRA_OEMAKE` to pass `MNTPOINT=yes`, instead of applying the patch.

Reply:
thanks! I'll fix it.